packages/core/src/postProcess/PostProcessEffectParameter.ts

```
@@ -0,0 +1,94 @@
+import { Color, MathUtil, Vector2, Vector3, Vector4 } from "@galacean/engine-math";
+import { Texture } from "../texture/Texture";
+
+/**
+ * Represents a parameter of a post process effect.
+ * @remarks
+ * The parameter will be mixed to a final value and be used in post process manager.
+ */
+export class PostProcessEffectParameter<T extends Number | Boolean | Color | Vector2 | Vector3 | Vector4 | Texture> {
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)